pyproject.toml

```diff
@@ -19,7 +19,8 @@ dependencies = [
     "fastapi>=0.95.0",
     "python-multipart",  # required for fastapi file uploads
     "uvicorn",
-    "ga4gh.vrs[extras]~=2.0.0a10",
+    "ga4gh.vrs[extras]~=2.0.0a11",
+    "sqlalchemy~=1.4.54",
```

Comment:
I'm not sure if we needed to add `sqlalchemy` to required dependencies (since not everyone will be using sqlalchemy). I think the optional `snowflake` dep handles this?

Reply:
@korikuzma I had the same thought but I think it's probably fine? (really there is only one user right now)

Either way, since it's named as an imported module it should be declared as an explicit dependency even if other dependencies will probably include it (in the sql groups or in main)

Reply:
Right, I think it would be nice to keep this in the `snowflake` optional dep to keep things lightweight

Reply:
It doesn't have to be now, but I'm at least going to make an issue to address in the future

Reply:
Commented in issue -- I assumed we were only using psycopg2 for postgres. @jsstevenson pointed out it's also used in snowflake. This makes sense now